### ISSUE TITLE
elasticsearch errors when parsing malformed date in otherProperties

### ISSUE THREAD

**Issue by erinspace** [fabianvf/scrapi#215]
When elasticsearch interprets one field as a particular type, say a date, it expects all other members of that field to be able to be parsed into the same type. Because of the lack of standards in the OAI PMH format for date, sometimes those date fields can't all be parsed as dates.

When dates are included, then the format changes, we get errors like this when harvesting:

```
elasticsearch.exceptions.RequestError: 
TransportError(400,u'MapperParsingException[failed to parse
 [otherProperties.properties.date]]; nested: 
MapperParsingException[failed to parse date field [c1992], 
tried both date format [dateOptionalTime], and timestamp number 
with locale []]; nested: IllegalArgumentException
[Invalid format: "c1992"]; ')
```

**Comment by fabianvf** [comment]
An easy solution would be to disable automated date format detecting, so we would have to manually specify when a field would be a date field in the elasticsearch mappings.

**Comment by fabianvf** [comment]
Another easy solution is to ignore malformed fields, which can be set in the global elasticsearch config.

**Comment by erinspace** [comment]
Closed in #334
